admin_guide/cluster_management.rst

```diff
@@ -354,37 +354,53 @@ Results:
-
-We can see that Citus used the real-time executor to run the count query. This executor fragments the query into constituent queries to run on each node and combines the results on the coordinator node. In the case of the second query (filtering by the distribution column ``id = $1``), Citus determined that it needed the data from just one node. Thus Citus chose the router executor to send the whole query to that node for execution. Lastly, we can see that the ``insert into foo select…`` statement ran with the insert-select executor which provides flexibility to run these kind of queries.  The insert-select executor must sometimes pull results to the coordinator and push them back down to workers.
-
-So far the information in this view doesn't give us anything we couldn't already learn by running the ``EXPLAIN`` command for a given query. However in addition to getting information about individual queries, the ``citus_stat_statements`` view allows us to answer questions such as "what percentage of queries in the cluster are run by which executors?"
+  -[ RECORD 1 ]-+----------------------------------------------
+  queryid       | -6844578505338488014
+  userid        | 10
+  dbid          | 13340
+  query         | SELECT count(*) FROM foo;
+  executor      | adaptive
+  partition_key |
+  calls         | 1
+  -[ RECORD 2 ]-+----------------------------------------------
+  queryid       | 185453597994293667
+  userid        | 10
+  dbid          | 13340
+  query         | insert into foo select generate_series($1,$2)
+  executor      | insert-select
+  partition_key |
+  calls         | 1
+  -[ RECORD 3 ]-+----------------------------------------------
+  queryid       | 1301170733886649828
+  userid        | 10
+  dbid          | 13340
+  query         | SELECT * FROM foo WHERE id = $1
+  executor      | adaptive
+  partition_key | 42
+  calls         | 1
+
+We can see that Citus uses the adaptive executor most commonly to run queries. This executor fragments the query into constituent queries to run on relevant nodes, and combines the results on the coordinator node. In the case of the second query (filtering by the distribution column ``id = $1``), Citus determined that it needed the data from just one node. Lastly, we can see that the ``insert into foo select…`` statement ran with the insert-select executor which provides flexibility to run these kind of queries.  The insert-select executor must sometimes pull results to the coordinator and push them back down to workers.
+
+So far the information in this view doesn't give us anything we couldn't already learn by running the ``EXPLAIN`` command for a given query. However in addition to getting information about individual queries, the ``citus_stat_statements`` view allows us to answer questions such as "what percentage of queries in the cluster are scoped to a single tenant?"
 
 .. code-block:: postgresql
 
-  SELECT executor, sum(calls)
```

Comment:
The type of executor no longer seems particularly interesting. What people want to know is whether the query is single-sharded.

Reply:
Makes sense